pyhyp/pyHyp.py

```diff
@@ -229,11 +249,17 @@ def __init__(self, comm=None, options=None, commonOptions=None, debug=False, ski
                 hypGrid.writeOutput()
 
                 # Save results
+                nDecimals = 4
+
                 self.results["name"][index] = optionName
                 self.results["outputFile"][index] = hypGrid.options["outputfile"]
-                self.results["gridRatio"][index] = float(hypGrid.hyp.hypdata.gridratio)
-                self.results["minQualityOverall"][index] = float(hypGrid.hyp.hypdata.minqualityoverall)
-                self.results["minVolumeOverall"][index] = float(hypGrid.hyp.hypdata.minvolumeoverall)
+                self.results["minQualityOverall"][index] = roundSig(
```

Comment:
Yes, I only round for stdout. I agree, I should probably move the rounding business to the print function and leave the raw number in `self.results`.

I don't use `floatfmt` because it only allows rounding to a certain digit after the decimal point. But my function rounds to significant figures, no matter where the decimal point is. I forgot exactly why I introduced it, but I believe the 'minQualityOverall' would either be a very long string or just 0. I think rounding to significant figures is better.

Let me know what you think.

Reply:
I moved the rounding into the print function.

Reply:
Thanks for separating the data and the stdout process. I think I understand this a bit better now, although I think a scientific notation, e.g., `floatfmt=".4e"`, would give you the consistent output needed. I understand it's not as nice and easy to read as the float/decimal format, and may not be desired in some places. 
I think using `roundSig` is probably fine, but might give the user the wrong idea. We should try to present these values carefully and make sure its not rounded too heavily. 4 seems a bit small but this depends on what is being presented. A number like `12345.6789` with 4 significant digits as used here would be rounded to `12350` using `roundSig`, but `.4f` would present the number as is, and using `.4e` as `1.2346e+04`.

I dont have many broad cases to test this on, and I want to get stuck on this as this is a minor thing, but perhaps @anilyil has any input or comments here. If not we can go ahead and merge.

Reply:
Somehow, the scientific notation did not even come to my mind...

I changed it locally but it does look worse. So not sure, I agree with your point about confusing the user. I am open to both solutions but I lean towards the scientific notation because it reduces ambiguity. 

Lets wait for @anilyil's opinion.

Reply:
@anilyil agrees with scientific, so lets do this then.